build/questbookLocalisationShift.py

```diff
@@ -0,0 +1,46 @@
+# Created by The Science Demon for trainvoi
+# Next time, don't have such a skill issue, thanks
+
+import json
+import os
+
+questbookPath = os.path.normpath(os.path.abspath(f"{__file__}/../../config/betterquesting/DefaultQuests.json"))
+
+def Setup():
+	questbook = GetQuestbook()
+
+	ShiftLocalisationKeys(questbook = questbook, location = "questDatabase:9")
+
+	input("Press Enter to exit...")
+
+def GetQuestbook():
+	with open(questbookPath, "r") as file:
+		return json.load(file)
+
+def ShiftLocalisationKeys(questbook: dict, location: str):
+	for quest in dict(questbook[location]):
+		questId = questbook[location][quest]["questID:3"]
+		print(f"Found quest with ID: {questId}")
+
+		wantedQuestTitel = "susy.quest.db.%s.%s" % (questId, "title")
+		wantedQuestDescription = "susy.quest.db.%s.%s" % (questId, "desc")
+
+		actualQuestTitel = questbook[location][quest]["properties:10"]["betterquesting:10"]["name:8"]
+		actualQuestDescription = questbook[location][quest]["properties:10"]["betterquesting:10"]["desc:8"]
+
+		if actualQuestTitel != wantedQuestTitel or actualQuestDescription != wantedQuestDescription:
+			questbook[location][quest]["properties:10"]["betterquesting:10"]["name:8"] = wantedQuestTitel
```

Comment:
This looks like it might accidentally overwrite a quest if this isn't done correctly? I can certainly imagine a quest replacing the title and description of another quest, such that when another quest attempts to find its correct title and description, it can't actually find it. Is this just impossible, or should this be mitigated?